src/components/quote/Quote.styles.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+import { fontSizes } from '../../theme/fonts'
+import { mediaQueries } from '../../theme/mediaQueries'
+
+export const StyledBlockquote = styled.blockquote`
+  text-align: center;
+  ${mediaQueries.sm} {
+    font-size: ${fontSizes.h4_mobile};
+  }
+`
```

Comment:
We should utilise the `SpaceProps` and `LayoutProps` passed in here - take a look at other files, but this can be done with the [`compose` helper](https://styled-system.com/api#compose) from `styled-system`.